Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.datasource.url=jdbc:mysql://localhost/mydb?useSSL=false&allowMultiQueries=true&serverTimezone=UTC\r\nspring.datasource.username=root\r\nspring.datasource.password=0511\r\nspring.datasource.driverClassName=com.mysql.cj.jdbc.Driver\r\nspring.jpa.show-sql=true\r\n\r\nserver.port=9090\r\ncouponService.url=http://localhost:8080/couponapi/coupons/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	
+++ b/src/main/resources/application.properties	
@@ -5,4 +5,4 @@
 spring.jpa.show-sql=true
 
 server.port=9090
-couponService.url=http://localhost:8080/couponapi/coupons/
+couponService.url=http://localhost:9091/couponapi/coupons/
